packages/react-api/src/hooks/useViewportFeatures.js

```diff
@@ -22,6 +23,14 @@ export default function useViewportFeatures(
   const viewport = useSelector((state) => state.carto.viewport);
   const [tiles, setTiles] = useState([]);
   const [isGeoJSONLoaded, setGeoJSONLoaded] = useState(false);
+  const debounceId = useRef(0);
```

Comment:
why 0 and not null?

Reply:
Nothing special. I just preferred 0 here. Null would be the same. I can change it if you want.